docs/source/index.rst

```diff
@@ -0,0 +1,34 @@
+.. Hyperledger Indy Node documentation master file, created by
+   sphinx-quickstart on Wed Nov  7 15:20:42 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Hyperledger Indy Node's documentation!
+=================================================
+
+
+.. toctree::
+   :maxdepth: 1
+
+   start-nodes.md 
+   add-node.md
+   auth_rules.md
+   ci-cd.md
+   cluster-simulation.md
+   helper-scripts.md
+   indy-file-structure-guideline.md
+   indy-running-locally.md
+   node-monitoring-tools-for-stewards.md
+   node-state-diagnostic-tools.md
+   pool-upgrade.md
+   requests-new.md
```

Comment:
The new format of requests is not yet implemented.
So, please refer to `requests.md` for now.